setup.cfg

```diff
@@ -4,7 +4,7 @@ paths = features/
 show_skipped = no
 
 [flake8]
-exclude = docs,.cache,.tox,*.egg-info
+exclude = docs,.cache,.tox,*.egg-info,.ropeproject
 
-[pytest]
+[tool:pytest]
```

Comment:
Why? Shouldn't this be `[pytest]` according to the [py.test documentation](http://doc.pytest.org/en/latest/customize.html#builtin-configuration-file-options)?